frontend/tests/unit/transforms/django-url-test.js

```diff
@@ -0,0 +1,18 @@
+import { setupTest } from 'outdated/tests/helpers';
+import { module, test } from 'qunit';
+
+module('Unit | Transform | django url', function (hooks) {
+  setupTest(hooks);
+
+  test('serializes', function (assert) {
+    const transform = this.owner.lookup('transform:django-url');
+    assert.strictEqual(
+      transform.serialize('foo.bar/test'),
+      'https://foo.bar/test'
+    );
+    assert.strictEqual(
+      transform.serialize('https://foo.bar/test'),
+      'https://foo.bar/test'
+    );
+  });
+});
```

Comment:
Drop

Reply:
done in https://github.com/adfinis/Outdated/pull/14/commits/7572607b01a33f53638e097b5113ac15566e417e